app/channels/user_channel.rb

```diff
@@ -1,11 +1,11 @@
 class UserChannel < ApplicationCable::Channel
   def subscribed
-    if !current_user.nil?
-      stream_from "users:#{current_user.id}"
-    else
-      reject
-    end
-  end
++    if current_user.nil?
+       reject
++    else
```

Comment:
Hi @melwhite1, thanks for opening this. The `+` should not be part of the code change. could you please check?. Also please update the pull request description with the issue number where you have `Fixes #0000`, It helps associate the PR to the issue and helps in closing once its merged. Thanks

Reply:
Hi! Ok, I think I fixed it!

Reply:
Hi @melwhite, I saw the new pull request and it is a little amiss. This looks closer than the other one, you could try removing the `+` at the beginning of line 3,5 and 6. If you can edit from here https://github.com/melwhite1/plots2/edit/patch-1/app/channels/user_channel.rb and commit this pull request should have the new changes. What do you think? Thanks for sticking with it

Reply:
HI @melwhite1, so you need any help here? Its so close and we are happy to help if you are stuck. Thanks